azure-pipelines.yml

```diff
@@ -13,6 +15,43 @@ steps:
 - script: pip install -U iotedgedev
   displayName: 'Install IoT Edge Dev Tool'
 
+- task: DotNetCoreCLI@2
+  displayName: 'Build C# projects and run unit tests'
+  inputs:
+    command: 'test' 
+    projects: '**/*.csproj'
+    arguments: '--collect:"XPlat Code Coverage" -- RunConfiguration.DisableAppDomain=true'
+
+- task: DotNetCoreCLI@2
+  displayName: "Install dotnet format"
+  inputs:
+    command: 'custom'
+    custom: 'tool'
+    arguments: 'install -g dotnet-format --version 5.0.211103 --add-source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json'
```

Comment:
Is there a reason to use specific version of `dotnet-format`?

Reply:
No, it was for testing and now it's removed :)